lib/floe/runner/container_runner/docker.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 module Floe
-  class Workflow
-    class Runner
+  class Runner
+    class ContainerRunner
       class Docker < Floe::Workflow::Runner
```

Comment:
Oh I thought you were getting rid of Floe::Workflow::Runner based on the above comment, nevermind then